app/viewmodels/govuk/CheckboxFluency.scala

```diff
@@ -49,16 +52,18 @@ trait CheckboxFluency {
 
     def apply(
                form: Form[_],
+               idPrefix: String,
                name: String,
                hint: Option[Hint],
                items: Seq[CheckboxItem],
                fieldset: Fieldset
              )(implicit messages: Messages): Checkboxes =
       Checkboxes(
         fieldset     = Some(fieldset),
+        idPrefix     = Some(idPrefix),
         name         = name,
         hint         = hint,
-        errorMessage = errorMessage(form(name)),
+        errorMessage = form.errors.headOption.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*)))),
```

Comment:
does this mean if there are multiple options we're only displaying one of them at a time?